File: Arkitektum.Orden/Models/Field.cs
namespace Arkitektum.Orden.Models
{
    public class Field
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public bool IsPersonalData { get; set; }

        public bool IsSensitivePersonalData { get; set; }

        public Dataset Dataset { get; set; }

        public int? DatasetId { get; set; }
    }
}

File: Arkitektum.Orden/ClientApp/components/modules/Field.js
export default {
    name: 'Field',
    props: {
        name: String,
        writeAccess: {
            type: Boolean,
            default: false
        }
    },
    data() {
        return {
            apiData: null,
            url: `/dataset/metadata/${this.datasetId}`
        }
    }
}